### PR TITLE
[SIMPLE_FORMS] feat: update attachment logic to handle form 20-10207

### PR DESCRIPTION
**Pull request by pennja**
## Summary

- This work adds logic to separate attachments when submitting form `20-10207`
- This work also adds test coverage around the new logic

## Related issue(s)

- [Error: Form 20-10207 Max Document size exceeded](https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/1610)

## Testing done

- [x] *New code is covered by unit tests*

## What areas of the site does it impact?

- Simple Forms API -> form 20-10207

## Requested Feedback

Any

### CODE REVIEW

**Review by Thrillberg — COMMENTED** [review]
Looking good! One comment inside, plus a test or two would be nice. Than this will be ready to go!

File: modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb [hunk 1]
```diff
@@ -71,6 +71,8 @@ def get_intents_to_file
 
       private
 
+      attr_reader :attachments
```
[the remark this answers is not in the document]
Comment (pennja) [reply modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb:74]: Definining the `attr_reader` after the `private` keyword mitigates any security risks of making the attachments variable global but I've gone ahead and changed the logic to use your suggestion given attachments wouldn't be needed anywhere else in the controller.

File: modules/simple_forms_api/app/models/simple_forms_api/vba_20_10207.rb [hunk 2]
```diff
@@ -135,35 +135,26 @@ def track_user_identity(confirmation_number)
...
+          medal_award_documents
+          pow_documents
+          terminal_illness_documents
+          vsi_documents
+        ].each do |doc_type|
+          next unless @data[doc_type]
+
+          confirmation_codes = @data[doc_type].pluck('confirmation_code')
+          attachments.concat(PersistentAttachment.where(guid: confirmation_codes).map(&:to_pdf))
+        end
```
Comment (Thrillberg) [inline modules/simple_forms_api/app/models/simple_forms_api/vba_20_10207.rb:152]: Nice refactor here!

**Review by Thrillberg — APPROVED** [review]
Looks great! Thanks for your excellent work on this!

**Review by rjohnson2011 — APPROVED** [review]

**Pull request merged**